### ISSUE TITLE
compilation problem

### ISSUE THREAD

**Issue by IngeniariusSoftware** [cqfn/eo#473]
code
```java
+package sandbox

+alias org.eolang.io.stdout

[args...] > app
  stdout "SOME" > @
```
`mvn clean coompile`

result: `SOME`

after that change "SOME" in code on "SOMETHING"
code
```java
+package sandbox

+alias org.eolang.io.stdout

[args...] > app
  stdout "SOMETHING" > @
```
`mvn compile`

result still: `SOME`

**Comment by andreoss** [comment]
@IngeniariusSoftware It have been introduce by https://github.com/cqfn/eo/pull/453. `mvn clean compile` works as expected?

**Comment by IngeniariusSoftware** [comment]
@andreoss yes, `mvn clean compile` works correctly, but I expect that after modifying eo file and run command `mvn compile` I got new result, but in reality I got the same

**Comment by yegor256** [comment]
@IngeniariusSoftware it's fixed, try version 0.20.0

**Comment by 0crat** [comment]
Job `gh:cqfn/eo#473` is not assigned, can't get performer
